src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -37,16 +37,21 @@ interface BuildLocator {
      * By default only successful builds are returned, call this method to include failed builds as well.
      */
     fun withAnyStatus() : BuildLocator
-
     fun withStatus(status: BuildStatus): BuildLocator
+
     fun withTag(tag: String): BuildLocator
 
-    fun withBranch(branch: String): BuildLocator
+    /**
+     * By default failed to start builds are filtered out, call this method to include them.
+     */
+    fun withAnyFailedToStart(): BuildLocator
```

Comment:
Yes, including this value into BuildStatus will make it a little less discoverable. However, adding 3 functions for such a rarely used option will make all other options in BuildLocator less discoverable.

And I've just checked, there is no 'Mark as successful' option in TeamCity UI for failed to start build.